docs/bibliography.bib

```diff
@@ -288,6 +299,29 @@ @article{stringer:1963
    pages = {89–107},
    doi = {10.1088/0368-3281/5/2/304}
 }
+@article{thompson:1995,
+title = {Wave behaviour near critical frequencies in cold bi-ion plasmas},
+journal = {Planetary and Space Science},
+volume = {43},
+number = {5},
+pages = {625-634},
+year = {1995},
+issn = {0032-0633},
+doi = {https://doi.org/10.1016/0032-0633(94)00197-Y},
+url = {https://www.sciencedirect.com/science/article/pii/003206339400197Y},
+author = {P. Thompson and M.K. Dougherty and D.J. Southwood},
+}
+@article{vincena:2013,
+author = {Vincena,S. T.  and Farmer,W. A.  and Maggs,J. E.  and Morales,G. J. },
+title = {Investigation of an ion-ion hybrid Alfv\'{e}n wave resonator},
```

Comment:
```suggestion
title = {Investigation of an ion-ion hybrid Alfvén wave resonator},
```

Reply:
It's always been my habit to escape accents, but if this is resolved with UTF, then so be it! :)